src/id/doujindesu/src/eu/kanade/tachiyomi/extension/id/doujindesu/DoujinDesu.kt

```diff
@@ -317,8 +323,53 @@ class DoujinDesu : ParsedHttpSource(), ConfigurableSource {
     // Search & FIlter
 
     override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val author = filters.filterIsInstance<AuthorFilter>().firstOrNull()?.state?.trim()
+        val group = filters.filterIsInstance<GroupFilter>().firstOrNull()?.state?.trim()
+        val series = filters.filterIsInstance<SeriesFilter>().firstOrNull()?.state?.trim()
+
+        // Author filter handling
```

Comment:
query search should be preferred first

Reply:
I meant, dont return author/group/series/etc individual requests if query isn't blank